kornia/augmentation/random_generator/_2d/crop.py

```diff
@@ -516,4 +518,4 @@ def center_crop_generator(
 
     _input_size = torch.tensor((height, width), device=device, dtype=torch.long).expand(batch_size, -1)
 
-    return dict(src=points_src, dst=points_dst, input_size=_input_size)
+    return dict(src=points_src, dst=points_dst, input_size=_input_size, output_size=size)
```

Comment:
```suggestion
    return dict(src=points_src, dst=points_dst, input_size=_input_size, output_size=_output_size)
```
Maybe more clear like this

Reply:
done